ci-operator/step-registry/openshift/microshift/infra/aws/ec2/openshift-microshift-infra-aws-ec2-commands.sh

```diff
@@ -15,8 +15,10 @@ trap 'save_stack_events_to_shared' EXIT TERM INT
 declare -A ami_map=(
   [us-west-2,x86_64,rhel-9.2]=ami-0d5b3039c1132e1b2
   [us-west-2,x86_64,rhel-9.3]=ami-04b4d3355a2e2a403
```

Comment:
Can we also update the 9.2 and 9.3 images? This should speed up the CI by installing less packages on `dnf upgrade all`